test/fixtures/settings.yml

```diff
@@ -348,3 +348,8 @@ attribute72:
   category: Setting::Provisioning
   default: nil
   description: 'Default host owner'
+attribute73:
+  name: local_boot_PXELinux
```

Comment:
You said you use downacase but there's still mixed case key here, I was hoping we get rid of this completely. I believe all or most of our settings are downcase only.